### ISSUE TITLE
SVG source breaks Image resizeMode repeat in Safari

### ISSUE THREAD

**Issue by dcalhoun** [necolas/react-native-web#1119]
**The problem**
Rendering `<Image resizeMode="repeat" />` with an SVG source does not repeat in macOS Safari and iOS Safari.

**How to reproduce**
Simplified test case: https://codesandbox.io/s/m7n8zm1kwj

Steps to reproduce:
1. Render `<Image resizeMode="repeat" />` with an SVG source

**Expected behavior**
In all browsers, the image repeats to fill the width and height of the container.

**Environment (include versions). Did this work in previous versions?**

* React Native for Web (version): 0.9.0
* React (version): 16.5.2
* Browser: macOS Safari 12, iOS Safari 12

**Comment by necolas** [comment]
Have you looked into why yet?

**Comment by dcalhoun** [comment]
Yes, it seems related to the calculation within `Image._getBackgroundSize`. I do not know why it is different at the present, but the "natural" dimensions are different in Safari.

https://github.com/necolas/react-native-web/blob/1f3a77dadad26c09dc3d854f07ffaf4f23f2417c/packages/react-native-web/src/exports/Image/index.js#L323

![rnw-background-sizing](https://user-images.githubusercontent.com/438664/46220762-c5c95800-c310-11e8-8a8e-20a46fff87e7.png)

**Comment by dcalhoun** [comment]
Bummer. Thank you for looking into it. I guess I'll just have to utilize a raster image for the time being.

**Comment by necolas** [comment]
I haven't tried altering the code, but you could try removing the height/width from the accessibility image to see if it changes the results you get in Safari: https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/exports/Image/index.js#L415. If it does, there might be something we can do. If not… :'(
